tsc v3.3.33 errors on dts @jxa/global-type

Hi there!

This might be a dumb question but when importing @jxa/global-type on my .ts file and then compiling I get this error:

```bash
node_modules/@jxa/global-type/src/index.d.ts:8:11 - error TS2403: Subsequent variable declarations must have the same type.  Variable 'Application' must be of type 'typeof Application', but here has type 'typeof Application & object'.

8     const Application: typeof Application_ & object;
            ~~~~~~~~~~~
Found 1 error.

```

seems like its causing conflict with this two definition type..

https://github.com/JXA-userland/JXA/blob/92b61a55975f4c9801fe6d488550ec7a5bb9b682/packages/%40jxa/global-type/src/index.d.ts#L8


Is it necessary to check that Application is also an object?

--- a/packages/@jxa/global-type/src/index.d.ts
+++ b/packages/@jxa/global-type/src/index.d.ts
@@ -5,7 +5,7 @@ import {
 } from "@jxa/types";
 
 declare global {
-    const Application: typeof Application_ & object;
+    const Application: typeof Application_;
     const Automation: typeof Automation_;
     const ObjectSpecifier: typeof ObjectSpecifier_;
 

